code/modules/chemistry/tools/hyposprays.dm

```diff
@@ -1,6 +1,6 @@
 var/global/list/chem_whitelist = list("antihol", "charcoal", "epinephrine", "insulin", "mutadone", "teporone",\
 "silver_sulfadiazine", "salbutamol", "perfluorodecalin", "omnizine", "synaptizine", "anti_rad",\
-"oculine", "mannitol", "penteticacid", "styptic_powder", "methamphetamine", "spaceacillin", "saline",\
+"oculine", "mannitol", "penteticacid", "styptic_powder", "methamphetamine", "spaceacillin", "saline", "isopropanol",\
```

Comment:
don't think it's a good fit for the medical whitelist

Reply:
Fair enough, forgot to remove it after removing the brute healing effect.